prow/prstatus/prstatus.go

```diff
@@ -201,11 +199,13 @@ func (da *DashboardAgent) HandlePrStatus(queryHandler PullRequestQueryHandler) h
 		// chance to validate whether the access token is consumable or not. If
 		// not, we invalidate the sessions and continue as if not logged in.
 		token, ok := session.Values[tokenKey].(*oauth2.Token)
-		var user *gogithub.User
+		var user *github.User
+		var botName string
 		if ok && token.Valid() {
-			goGithubClient := ghclient.NewClient(token.AccessToken, false)
+			goGithubClient := github.NewClient(func() []byte { return []byte(token.AccessToken) }, githubEndpoint)
 			var err error
-			user, err = queryHandler.GetUser(goGithubClient)
+			botName, err = goGithubClient.BotName()
```

Comment:
Same here, `githubClient` instead of `goGithubClient`.

Reply:
this too